<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpikonMainWindow</class>
 <widget class="QMainWindow" name="EpikonMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EpikonMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="gameView">
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="l_you">
          <property name="text">
           <string>You</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="l_enemy">
          <property name="text">
           <string>Enemy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLCDNumber" name="lcd_you"/>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="lcd_enemy"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="l_gameinfo">
          <property name="text">
           <string>Game Info</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="l_displaygameinfo">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionLocal_LAN_Game"/>
    <addaction name="actionInternet_Game"/>
   </widget>
   <widget class="QMenu" name="menuOther_Stuff">
    <property name="title">
     <string>Other Stuff</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Epikon"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuOther_Stuff"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionLocal_LAN_Game">
   <property name="text">
    <string>Local LAN Game</string>
   </property>
  </action>
  <action name="actionInternet_Game">
   <property name="text">
    <string>Internet Game</string>
   </property>
  </action>
  <action name="actionAbout_Epikon">
   <property name="text">
    <string>About Epikon</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
